web/src/main/ui/app/components/flows-new/edit-flow/mastering/merging/ui/add-merge-strategy-dialog.component.html

```diff
@@ -36,14 +36,14 @@ <h1 mat-dialog-title>{{getDialogTitle()}}</h1>
       <div [formGroupName]="i" class="source-weights-group">
         <mat-form-field>
           <mat-label>Source Name</mat-label>
-          <input [class]="'source-weights-source-' + i" matInput formControlName="source"/>
+          <input [class]="'source-weights-name-' + i" matInput formControlName="source"/>
         </mat-form-field>
         <mat-icon></mat-icon>
         <mat-form-field>
           <mat-label>Weight</mat-label>
           <input [class]="'source-weights-weight-' + i" matInput formControlName="weight"/>
         </mat-form-field>
-        <button class="remove-source-weight-btn" mat-icon-button (click)="onRemoveSourceWeight(i)">
+        <button [class]="'remove-source-weight-btn-' + i" mat-icon-button (click)="onRemoveSourceWeight(i)">
```

Comment:
I think we should remove the square brackets on the class? should be:
<button class="'remove-source-weight-btn-' + i" mat-icon-button (click)="onRemoveSourceWeight(i)">

This cause a css/styling bug on the minus button on merge strategy dialog, the minus button has a square background:
<img width="525" alt="Screen Shot 2019-04-11 at 9 51 48 AM" src="https://user-images.githubusercontent.com/7096668/55976239-8cb75c00-5c40-11e9-8314-5ab6f73bca1e.png">

It should look like:
<img width="521" alt="Screen Shot 2019-04-11 at 9 52 08 AM" src="https://user-images.githubusercontent.com/7096668/55976285-a3f64980-5c40-11e9-88a3-86a04ab27db7.png">

And the class name should be: remove-merge-strategy-source-weight-btn, just to be consistent with the add-merge-strategy-source-weight-btn

Reply:
@ayuwono Fixed and updated wiki.  Using IDs now for the source weight buttons for both options and strategies.